translations/bnpchs-2018-final-bermain-dengan-bentuk.html

```diff
@@ -0,0 +1,91 @@
+<h3>Description</h3>
+<p style="text-align: justify;">
+  Jelix is indeed naughty! This time he did something mischievous.
+  Jelix did not tidy up the legos he had played and left them on the floor.
+  Seeing this, Jelix's parents scolded Jelix and ordered him to tidy up immediately.
+</p>
+
+<p style="text-align: justify;">
+  We can represent those legos in a two-dimensional grid where each cell contains a lego.
+  Formally if there is a lego located in $(x, y)$, then the legos adjacent to it are in $(x+1, y)$, 
+  $(x-1, y)$, $(x, y+1)$, and $(x, y-1)$.
+  Two legos are alike if they have the same color.
+</p>
+
+<p style="text-align: justify;">
+  When he wants to tidy his legos, he saw that among those scattered legos, legos that are alike 
+  form strange shapes.
+  A strange shape is defined as two-dimensional figure consisting of alike legos that are adjacent 
+  horizontally or vertically.
+</p>
+
+<p style="text-align: justify;">
+  A strange shape can be found several times in different orientations by rotating them 90 degrees.
+  The following is an example of some shapes and all of their orientations.
+  Shapes colored in one color are the same shape.
+  <!-- ^ unsure -->
+</p>
+
+<img src="bnpchs-2018-final-bermain-dengan-bentuk-example.png" style="display:block; margin-left: auto;
+ margin-right: auto;">
+
+<p style="text-align: justify;">
+  Jelix wants to find the number of strange shapes that he can find from the scattered legos.
+  But due to being rushed to tidy them, he ask your help as a reliable coder.
+  Don't worry, Jelix has given you a rough picture of the legos' positions and colors!
+</p>
+
+
+<h3>Input Format</h3>
+<p style="text-align: justify;">
+  The first line consists of an integer $T$ ($1 \leq T \leq 20$) that determines the number 
+  of test cases.
+</p>
+<p style="text-align: justify;">
+  For each test case, the first line consists of two integers $R$ and $C$ ($1 \leq R, C \leq 50$) 
+  that determine the size of the two-dimensional grid representing Jelix's legos.
+</p>
+<p style="text-align: justify;">
+  The next $R$ lines consist of $C$ integers $A_y$ ($1 \leq A_y \leq 100$) that determine the color 
+  of each legos on the grid.
+</p>
+
+<h3>Output Format</h3>
+<p style="text-align: justify;">
+  For each test case, output <code>Case #X: Y</code>. <code>X</code> represents the case number 
```

Comment:
```suggestion
  For each test case, output <code>Kasus #X: Y</code>, where <code>X</code> represents the case number 
```

No need to translate output
and separate two inline codes with a word for better clarity